dev-lua/lua-openssl/lua-openssl-0.7.3.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils toolchain-funcs
+
+DESCRIPTION="OpenSSL binding for Lua"
+HOMEPAGE="https://github.com/zhaozg/lua-openssl"
+LUA_AUX_COMMIT="b56f6937096acea34ddf241ec7ea08ac52414d18"
+LUA_COMPAT_COMMIT="daebe77a2f498817713df37f0bb316db1d82222f"
+SRC_URI="https://github.com/zhaozg/lua-openssl/archive/${PV}.tar.gz -> ${P}.tar.gz
+	https://github.com/zhaozg/lua-auxiliar/archive/${LUA_AUX_COMMIT}.tar.gz -> ${PN}-aux-${LUA_AUX_COMMIT}.tar.gz
+	https://github.com/keplerproject/lua-compat-5.3/archive/${LUA_COMPAT_COMMIT}.tar.gz -> ${PN}-compat-${LUA_COMPAT_COMMIT}.tar.gz"
+
+LICENSE="MIT openssl PHP-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="libressl luajit"
+
+RDEPEND="
+	luajit? ( dev-lang/luajit:2 )
+	!luajit? ( >=dev-lang/lua-5.1:0 )
+	!libressl? ( dev-libs/openssl:0=[-bindist] )
+	libressl? ( dev-libs/libressl:0= )
+	!dev-lua/luaossl
+	"
+DEPEND="${RDEPEND}
+	virtual/pkgconfig"
+
+src_unpack() {
+	unpack "${P}.tar.gz"
+	pushd "${WORKDIR}/${P}/deps" || die
+	unpack "${PN}-aux-${LUA_AUX_COMMIT}.tar.gz"
+	unpack "${PN}-compat-${LUA_COMPAT_COMMIT}.tar.gz"
+	rmdir auxiliar lua-compat
+	mv "lua-auxiliar-${LUA_AUX_COMMIT}" auxiliar
+	mv "lua-compat-5.3-${LUA_COMPAT_COMMIT}" lua-compat
+	popd
+}
+
+src_prepare() {
+	default
+	use luajit && LUAV=luajit || LUAV=lua
+}
+
+src_compile() {
+	local pkgconfig=$(tc-getPKG_CONFIG)
```

Comment:
Alternatively, you can `tc-export PKG_CONFIG`, i think. Then it might be unnecessary to explicitly pass it to `emake`.